### ISSUE TITLE
Date format error on Windows (Powershell)

### ISSUE THREAD

**Issue by flacle** [orson1282/arubabank_api#8]
Entering dates as given in the documented format YYYY-MM-DD or even DD/MM/YYYY for the `-t` and `-f` flags gives the following error:
```
Traceback (most recent call last):
  File "...\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "...\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "...\arubabank_api\arubabank\__main__.py", line 154, in <module>
    main()
  File "...\arubabank_api\arubabank\__main__.py", line 149, in main
    transactions = get_transactions(args, api)
  File "...\arubabank_api\arubabank\__main__.py", line 39, in get_transactions
    return api.get_transactions(
  File "...\arubabank_api\arubabank\__init__.py", line 209, in get_transactions
    first_date = dt.strptime(
  File "...\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "...\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '06/14/2022' does not match format '%Y-%m-%d'
```

I am not sure if this is platform-specific.

**Comment by orson1282** [comment]
Thanks @flacle 
I'm guessing Aruba Bank updated the way their API returned the date. Fixed in 0.2.2. Give it a try

**Comment by flacle** [comment]
Tested, all good (y) - Awesome @orson1282
